.github/workflows/develop.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8"]
-        poetry-version: [1.1.11]
+        python-version: ["3.9"]
```

Comment:
All of our airflow and perfai runs on Python 3.8 at the moment, so we need to test on that

Reply:
This version will be released on 3.0.0 so we can test and choose whether to upgrade or not!

Reply:
Sure, we should update to 3.0.0, but you won't be able to install awesome-object-store 3.0.0 until we upgrade our docker containers to use python 3.9 👍

Reply:
I have a jira ticket to upgrade to 3.9, i should do that soon

Reply:
nice. so can you give me approval 🙏

Reply:
I just realised, we would also need to upgrade the version of Python in our dags and ndpoint repos, otherwise the versions will conflict :/